best_practices/creating-the-project.rst

```diff
@@ -8,35 +8,12 @@ In the past, Symfony projects were created with `Composer`_, the dependency mana
 for PHP applications. However, the current recommendation is to use the **Symfony
 Installer**, which has to be installed before creating your first project.
 
-Linux and Mac OS X Systems
```

Comment:
What do you think about moving it before the "Creating the Blog Application" headline?